Src/FluentAssertions/Equivalency/IEquivalencyAssertionOptions.cs

```diff
@@ -53,7 +53,7 @@ public interface IEquivalencyAssertionOptions
         /// <summary>
         /// Gets an ordered collection of Equivalency steps how a subject is compared with the expectation.
         /// </summary>
-        IEnumerable<IEquivalencyStep> UserEquivalencySteps { get; }
+        IEnumerable<IEquivalencyStep> GetUserEquivalencySteps(ConversionSelector conversionSelector);
```

Comment:
Yes, now I think that we need to change the public API because it's design was flawed.
On the other hand, the change is of the kind I like the most:

- impact on a tiny part of the user base
- compile-time
- trivial to fix

Nonetheless, I will look into it over the upcoming days, maybe so idea will emerge. I would not count on that though.

Reply:
Just do it. That could might be public, but it's really just internal stuff and never mentioned in any of the extensibility docs.

Reply:
Thanks, I won't worry about it then.